index.js

```diff
@@ -1,13 +1,17 @@
 'use strict';
-var chalk = require('chalk');
+const chalk = require('chalk'),
+    r = chalk.red,
+    yB = chalk.yellow.bold,
+    y = chalk.yellow;
```

Comment:
This makes the code a lot less readable. All the changes in this file should be reverted.

Reply:
Thanks for your feedback @sindresorhus 
I'll revert it asap ^^

Reply:
Fixed in https://github.com/yeoman/yeoman-welcome/pull/2/commits/bb4fd964b553a6d525bced121216ee4be61e6169